ldap/servers/plugins/replication/repl5_connection.c

```diff
@@ -1222,7 +1228,48 @@ conn_connect_with_bootstrap(Repl_Connection *conn, PRBool bootstrap)
 
         /* override the default timeout with the specified timeout */
         ldap_set_option(conn->ld, LDAP_OPT_NETWORK_TIMEOUT, &conn->timeout);
-        /* We've got an ld. Now bind to the server. */
+
+        /* if we have TLS uris, set them now */
+#ifdef LDAP_OPT_X_TLS_URIS
```

Comment:
I'm looking at the openldap source and I can't find a reference to this. Can you cite the docs about this flag?

Reply:
The flag is based on OpenSSL provider support added to openldap here:

https://bugs.openldap.org/show_bug.cgi?id=10149
https://github.com/openldap/openldap/pull/11